projects/components/core/src/button.ts

```diff
@@ -0,0 +1,10 @@
+import { ThemePalette } from '@angular/material/core';
+
+export interface IPsButton {
+  label?: string;
+  type: 'raised' | 'stroked' | 'icon';
```

Comment:
Would be useful to also support tyes 'flat' and 'default' (not sure about the name, I mean a plain mat-button)